pom.xml

```diff
@@ -571,6 +571,62 @@
           </execution>
         </executions>
       </plugin>
+
+      <plugin>
+        <artifactId>maven-compiler-plugin</artifactId>
+        <executions>
+          <execution>
+            <id>test-service-helper</id>
+            <phase>test-compile</phase>
+            <goals>
+              <goal>testCompile</goal>
+            </goals>
+            <configuration>
+              <compileSourceRoots>
+                <compileSourceRoot>src/test/classpath/servicehelper</compileSourceRoot>
+              </compileSourceRoots>
+              <outputDirectory>${project.build.directory}/classpath/servicehelper</outputDirectory>
+            </configuration>
+          </execution>
+          <execution>
```

Comment:
The reason that there is a second clause here is that there is a pattern in the test code of creating a new classloader and adding a directory such as the  _${project.build.directory}/classpath/servicehelper_ above to it's classpath. If I had a single folder, all the test classloaders would have combined classpaths and in order to seperate them, the smallest chunk of XML that I can duplicate is a an <execution>.